timemanagercontrol.py

```diff
@@ -19,6 +19,8 @@
 from conf import *
 from tmlogging import info, warn, error, log_exceptions
 
+# in python script opnemen
+import pydevd
```

Comment:
Is this important to add ? Makes travis fail, adds a new dependency, and I don't see where it is used